docs/guide/protocol-and-quick-start.md

```diff
@@ -0,0 +1,143 @@
+The simulation server will start Webots with the simulation requested by the client.
+Then, the simulation server will send the WebSocket URL of Webots, so that the client can communicate directly with Webots.
+If the WebSocket connection to the simulation server is closed or broken, the simulation server will close Webots.
+If Webots quits, the simulation server will notify the web client and close the connection with it.
+
+The session server regularly queries the simulation servers to monitor their load.
+If a simulation server becomes overloaded, then, the session server will consider it is not available any more and will not offer it to web clients.
+
+%figure "Web simulation protocol sequence diagram"
+%chart
+sequenceDiagram
+  participant U as User
+  participant C as Web Client
+  participant W as Web Server
+  participant G as GitHub Repository
+  participant SE as Session Server
+  participant SS1 as Simulation Server 1
+  participant SW1 as Webots
+  participant SS2 as Simulation Server 2
+
+  SE->>SS1: Check server availability
+  activate SS1
+    SS1-->>SE: Return machine load
+  deactivate SS1
+  SE->>SS2: Check server availability
+  activate SS2
+    SS2-->>SE: Return machine load
+  deactivate SS2
+
+  activate C
+    C->>W: Load web page
+    activate W
+      W->>SE: Check server availability
+      activate SE
+        SE-->>W: 0 or 1
+      deactivate SE
+      W-->>C: Return web page content
+    deactivate W
+    U->>C: Start simulation
+    Note left of C: webots-streaming web component
+    C->>SE: Open simulation session
+    activate SE
+      SE-->>C: Return simulation server web socket URL
+    deactivate SE
+    C->>SS1: Start simulation
+    activate SS1
+      SS1->>G: Clone simulation project
+      activate G
+        Note right of G: git clone repository
+        G-->>SS1: Simulation project
+      deactivate G
+      SS1->>SW1: Start Webots
+      activate SW1
+        SS1-->>C: Return Webots web socket URL
+      deactivate SS1
+      C->>SW1: Register client
+      SW1->>C: Send simulation worlds status
+      U->>C: Run simulation
+      C->>SW1: Change simulation running mode
+      loop Each simulation step
+        SW1->>C: Send simulation world status
+      end
+      U->>C: Reset simulation
+      C->>SW1: Reset simulation
+      loop Each simulation step
+        SW1->>C: Send simulation worlds status
+      end
+      C-->>SS1: {Client disconnected}
+    deactivate C
+    activate SS1
+      SS1->>SW1: Kill Webots
+    deactivate SW1
+  deactivate SS1
+%end
+%end
+
+### Quick Start
+This section gives a simple step-by-step guide on how to start a streaming server with one session and one simulation server.
+We assume you use Ubuntu 20.04 or newer.
+
+First, you need to install dependencies:
+```bash
+sudo apt install subversion docker python3-tornado python3-pynvml
```

Comment:
the requirement from `Linux Optionnal Dependencies` are not the same as here.

Reply:
I have added a link to the webots wiki linux web service dependencies instead.